protocol/0093-CFUT-product_builtin_capped_future.md

```diff
@@ -0,0 +1,63 @@
+# Built-in [Product](./0051-PROD-product.md): Capped future
+
+This built-in product provides capped futures contracts which are similar in architecture to fixed-expiry [cash settled futures](./0016-PFUT-product_builtin_future.md) with the main difference being that prices in the market are only allowed to fluctuate within the `[0, max_price]` range. This applies to all prices in the market, that's is:
+
+- order (and thus trade) prices,
+- mark prices,
+- final settlement price.
+
+The `max_price` is set at market creation time and it's immutable throughout the market's lifecycle.
+
+## 1. Product parameters
+
+Capped future should have all the product parameters and behaviour of a standard fixed-expiry cash settled [future](./0016-PFUT-product_builtin_future.md#1-product-parameters) as well as: 
+
+1. `max_price`: specifies the price cap for the market, an integer understood in the context of market decimal places,
+1. `binary_settlement`: if set to `true` settlement price other than `0` or `max_price` will be ignored,
+1. `fully_collateralised`: if set to `true` the [fully collateralised margin mode](./0019-MCAL-margin_calculator.md#fully-collateralised) is used.
+
+Validation: `max_price` > 0.
+
+## 2. Additional considerations
+
+### 2.1 Order price validation
+
+In order to aid smooth operation of the market and safeguard against user errors or manipulation attempts order prices should be validated so that the maximum price of any order is `max_price`. Peg orders should be validated so that if the resulting price would be higher at any point it gets temporarily capped at `max_price`.
+
+### 2.2 Mark price validation
+
+Irrespective of how the mark price is obtained in the market, the final value to be applied used in the [mark-to-market](./0003-MTMK-mark_to_market_settlement.md) calculation must be capped so that `mark_price = min(max_price, mark_price_candidate)`.
```

Comment:
Capping is inconsistent with ignoring out of bounds values below. Given we changed those recently I wanted to check we definitely want to cap here?